libraries/chain/include/eosio/chain/hotstuff/finalizer.hpp

```diff
@@ -0,0 +1,149 @@
+#pragma once
+#include <eosio/chain/hotstuff/hotstuff.hpp>
+#include <eosio/chain/hotstuff/finalizer_policy.hpp>
+#include <eosio/chain/fork_database.hpp>
+#include <fc/crypto/bls_utils.hpp>
+#include <fc/io/cfile.hpp>
+#include <fc/reflect/reflect.hpp>
+#include <fc/crypto/sha256.hpp>
+#include <compare>
+#include <utility>
+
+namespace eosio::chain {
+   // ----------------------------------------------------------------------------------------
```

Comment:
This is a new file, can we add a comment(s) that helps define what changes go into finalizer.hpp/finalizer.cpp by explaining the purpose or role of finalizer structs? This would help future developers understand what changes should go here vs other files like the controller.

Reply:
added.